django_tasks/backends/database/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+from .models import DBTaskResult
+
+
+@admin.register(DBTaskResult)
+class DBTaskResultAdmin(admin.ModelAdmin):
+    list_display = ("id", "task_path", "status", "priority", "queue_name")
+    list_filter = ("status", "priority", "queue_name")
+
+    def has_add_permission(self, request, obj=None):
```

Comment:
Issue: I'd suggest also adding `has_delete_permission` and `has_change_permission`:

```python
    def has_delete_permission(self, request, obj=None):
        return False

    def has_change_permission(self, request, obj=None):
        return False
```

This removes a few buttons, which cleans up the UI a little.